### PR TITLE
More tof compressors on FLPs (+option paranoid if requested)

### PR DESCRIPTION
**Pull request by noferini**
Two changes (both validated in TOF standalone runs, cosmics)
- Move from 3(6) to 11(11) compressors in tof-full-qcmn and tof-compressors workflows
- add workflow with paranoid option

### CODE REVIEW

File: scripts/tof-full-qcmn-paranoid.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```
Comment (knopers8) [inline scripts/tof-full-qcmn-paranoid.sh:1]: please add the new script to `scripts/generate-all-dpl-workflows.sh` to make it run for each new release?

Comment (noferini) [reply scripts/tof-full-qcmn-paranoid.sh:1]: > please add the new script to `scripts/generate-all-dpl-workflows.sh` to make it run for each new release?

Done. Thanks

**Review by knopers8 — APPROVED** [review]

File: scripts/tof-full-qcmn-paranoid.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+o2-qc --config $QC_GEN_CONFIG_PATH --remote -b --o2-control $WF_NAME
+
+# add the templated QC config file path
+ESCAPED_QC_FINAL_CONFIG_PATH=$(printf '%s\n' "$QC_FINAL_CONFIG_PATH" | sed -e 's/[\/&]/\\&/g')
+sed -i /defaults:/\ a\\\ \\\ "${QC_CONFIG_PARAM}":\ \""${ESCAPED_QC_FINAL_CONFIG_PATH}"\" workflows/${WF_NAME}.yaml
+
+# find and replace all usages of the QC config path which was used to generate the workflow
+ESCAPED_QC_GEN_CONFIG_PATH=$(printf '%s\n' "$QC_GEN_CONFIG_PATH" | sed -e 's/[]\/$*.^[]/\\&/g');
+sed -i "s/""${ESCAPED_QC_GEN_CONFIG_PATH}""/{{ ""${QC_CONFIG_PARAM}"" }}/g" workflows/${WF_NAME}.yaml tasks/${WF_NAME}-*
```
Comment (knopers8) [inline scripts/tof-full-qcmn-paranoid.sh:50]: @noferini do you intend to use the `remote` of the workflow part coming from this script?

Comment (noferini) [reply scripts/tof-full-qcmn-paranoid.sh:50]: Indeed I copied the same of the tof-full-qcmn.sh
We want to have just one remote merger for both.
I didn't know what to do in such a case

Comment (noferini) [reply scripts/tof-full-qcmn-paranoid.sh:50]: I guess it is redundant, let me know what shall I do (or if you want to change it by yourself)

Comment (knopers8) [reply scripts/tof-full-qcmn-paranoid.sh:50]: since it is redundant, we can remove it, a PR is coming

Comment (noferini) [reply scripts/tof-full-qcmn-paranoid.sh:50]: ok, thanks

**Pull request merged**